html-forms/index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>HTML Forms</title>
+</head>
+<body>
+  <h1>Coding Form!</h1>
+  <form action="">
+    <div>Email
+      <input type="text" name="email"/>
+    </div>
+    <div>Why is coding cool?</div>
```

Comment:
This should be a `label` element. Make sure you use a `for` attribute with a value that matches the `id` of the form control element this is labelling!